### PR TITLE
Change logic of testing for libunwind on OSX and ARM

### PR DESCRIPTION
**Pull request by NanoAkron**
Change from negative to positive tests

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -273,14 +273,19 @@ endif()
 
 add_definitions("-DBLOCKCHAIN_DB=${BLOCKCHAIN_DB}")
 
-find_package(Libunwind)
 # Can't install hook in static build on OSX, because OSX linker does not support --wrap
 # On ARM, having libunwind package (with .so's only) installed breaks static link.
-if(LIBUNWIND_FOUND AND NOT (STATIC AND (APPLE OR ARM)))
-  set(DEFAULT_STACK_TRACE ON)
-else()
+if(STATIC OR APPLE OR ARM)
```
Comment (NanoAkron) [inline CMakeLists.txt]: I may need to change this if we want it to build with libunwind on ARM/OSX in non-static mode

Comment (NanoAkron) [inline CMakeLists.txt]: OK, there's an easy way to do this. Will fix shortly.

**Comment by NanoAkron** [comment]
@kenshi84 Would you mind testing once more? Should now work with non-static compiles on ARM as well as any on OSX

**Comment by moneromooo-monero** [comment]
ok

**Comment by kenshi84** [comment]
@NanoAkron Also works! Thank you :)

**Comment by vickio** [comment]
This pr fixes non-static builds for me as well on macOS 10.12.

**Pull request merged**